src/care_radiology/api/webhooks.py

```
@@ -114,3 +135,173 @@ def save_webhook(self, request):
+
+            # Step 8a: Get cached access token or fetch new one
+            access_token = cache.get("admin_access_token")
+
+            if not access_token:
+                logger.info(
+                    "[MPPS] Access token not in cache, fetching new one..."
+                )
+                login_url = "http://localhost:9000/api/v1/auth/login/"
+                login_payload = {"username": "admin", "password": "admin"}
+
+                login_response = requests.post(login_url, json=login_payload)
```

Ruff (0.15.18)

[error] Probable use of `requests` call without timeout

(S113)